test/algorithms/buffer/buffer_linestring.cpp

```diff
@@ -326,6 +345,12 @@ void test_all()
         using bg::strategy::buffer::end_round;
         test_one<linestring, polygon>("issue_1084", issue_1084, join_round(10), end_round(10), 13200.83, 10.0);
     }
+    {
+        using bg::strategy::buffer::join_miter;
+        using bg::strategy::buffer::end_flat;
+        test_one<linestring, polygon>("issue_1250", issue_1250, join_miter(5), end_flat(), 8.39277, 0.5);
+        test_one<linestring, polygon>("issue_1250_rev", issue_1250_rev, join_miter(5), end_flat(), 8.39277, 0.5);
```

Comment:
This cases is fixed. See the issue for a picture.
It was surprisingly simple case - and therefore I was surprised that it was failing, and similar other cases were not failing. The cause was that it increased the next-index, because there was only one turn in the outer ring, and therefore it pointed to the wrong next turn.